src/sidebar/components/annotation-header.js

```diff
@@ -24,6 +24,9 @@ function AnnotationHeader({
 }) {
   const annotationLink = annotation.links ? annotation.links.html : '';
   const replyPluralized = !replyCount || replyCount > 1 ? 'replies' : 'reply';
+  // NB: `created` and `updated` are strings, not `Date`s
+  const hasBeenEdited =
+    annotation.updated && annotation.created !== annotation.updated;
```

Comment:
I have a suggestion for a subsequent iteration - we might want to alter the comparison so that if the created and updated dates are within a short time of one another, eg. due to a user editing immediately after posting, we don't show the "(edited ...)" notice.